scripts/zombienet/polimec-paseo-local.toml

```diff
@@ -4,7 +4,7 @@ provider = "native"
 
 [relaychain]
 default_command = "polkadot"
-chain_spec_path = "./scripts/zombienet/relay-chain-specs/paseo-local.plain.json"
+chain = "rococo-local"
```

Comment:
why use rococo instead of paseo if Polimec expects paseo?

Reply:
Because even with the latest Paseo plain chain_spec (src: https://github.com/paseo-network/runtimes/blob/main/chain-specs/paseo-local.plain.json) - it doesn't work.

![image](https://github.com/user-attachments/assets/60d1b502-f0c2-4c5c-8a8f-7cc7a8093581)